src/lib.rs

```diff
@@ -275,6 +286,8 @@ impl ValueEnum for Shell {
             Self::Elvish => clap::builder::PossibleValue::new("elvish"),
             Self::Fig => clap::builder::PossibleValue::new("fig"),
             Self::Fish => clap::builder::PossibleValue::new("fish"),
+            #[cfg(feature = "man-page")]
+            Self::ManPage => clap::builder::PossibleValue::new("man"),
```

Comment:
I'm torn on what this should be, for example:
```suggestion
            Self::ManPage => clap::builder::PossibleValue::new("man-page"),
```